tests/charts/api_tests.py

```diff
@@ -435,6 +436,11 @@ def test_create_chart(self):
         """
         Chart API: Test create chart
         """
+        dashes = (
+            db.session.query(Dashboard)
```

Comment:
nit: we can just select the id's here `db.session.query(Dashboard.id)`

Reply:
done